ddl/ddl_worker.go

```diff
@@ -285,8 +285,8 @@ func (w *worker) finishDDLJob(t *meta.Meta, job *model.Job) (err error) {
 		}
 	}
 	switch job.Type {
-	case model.ActionRestoreTable:
-		err = finishRestoreTable(w, t, job)
+	case model.ActionRecoverTable:
```

Comment:
What about the compatibility, if the job is pushed by old version tidb, the model.ActionRestoreTable job type will be treated as invalid job?